Framework/Algorithms/src/PolarizationCorrections/HeliumAnalyserEfficiency.cpp

```diff
@@ -37,10 +37,10 @@ auto constexpr OUTPUT_WORKSPACE{"OutputWorkspace"};
 auto constexpr OUTPUT_FIT_CURVES{"OutputFitCurves"};
 auto constexpr OUTPUT_FIT_PARAMS{"OutputFitParameters"};
 auto constexpr SPIN_STATES{"SpinStates"};
-auto constexpr PD{"GasPressureTimesCellLength"};
-auto constexpr PD_ERROR{"GasPressureTimesCellLengthError"};
-auto constexpr START_LAMBDA{"StartLambda"};
-auto constexpr END_LAMBDA{"EndLambda"};
+auto constexpr PXD{"PXD"};
+auto constexpr PXD_Error{"PXDError"};
+auto constexpr StartX{"StartX"};
```

Comment:
Our naming convention for variables that are being used as constants in Mantid is to capitalise and separate words with an underscore, so please could the `StartX` and `EndX` variable names be re-named to `START_X` and `END_X`? It's only the variable name that needs to be changed, the string values that are assigned to them are fine.

Reply:
Sorry, I've also noticed that `PXD_Error` also needs to be changed to `PXD_ERROR` to fit with our naming convention for constants. So please could we make the following changes to variable names:

`PXD_Error{"PXDError"};` -> `PXD_ERROR{"PXDError"};`
`StartX{"StartX"};` -> `START_X{"StartX"};`
`EndX{"EndX"};` -> `END_X{"EndX"};`